applications_tests/gls_sharp_navier_stokes_2d/couette2d_gls_poisson.output

```diff
@@ -7,26 +7,26 @@ Running on 1 MPI rank(s)...
 Steady iteration :        1/3
 *****************************************************************
 div u : 0
-L2 error velocity : 0.00418651 L2 error pressure: 0
```

Comment:
What happened here? Why is the error so big all of a sudden?

Reply:
I changed the omega by mistake, I'll fix it